src/components/Pricing/Pricing.tsx

```diff
@@ -37,84 +38,86 @@ interface PlanData {
-            '10 projects',
-            '7 year data retention',
-            'Email support',
-            'Pay only for what you use',
-            <>
-                <span className="opacity-60 text-sm">* Included with any product subscription</span>
-            </>,
-        ],
-    },
-    {
-        title: 'Teams',
-        price: '$450',
-        features: [
-            'Generous free tier on all products',
-            <>
-                <span className="relative">
-                    Includes team features{' '}
-                    <Tooltip
-                        content={() => (
-                            <>
-                                <h3 className="mb-1 text-base">Team features</h3>
-                                <ul className="list-none p-0 divide-y divide-light dark:divide-dark">
-                                    <li className="py-1.5 text-sm">Verified events</li>
-                                    <li className="py-1.5 text-sm">Comments on dashboards and insights</li>
-                                    <li className="py-1.5 text-sm">Data taxonomy (tags and descriptions)</li>
-                                </ul>
-                            </>
-                        )}
-                        tooltipClassName="max-w-sm"
-                    >
-                        <span className="relative -top-px">
-                            <IconInfo className="inline-block w-4 h-4" />
-                        </span>
-                    </Tooltip>
-                </span>
-            </>,
-            'Unlimited projects',
-            '7 year data retention',
-            'Priority support',
-            'Pay only for what you use',
-        ],
-        CTAText: 'Get started',
-    },
-    {
-        title: 'Enterprise',
-        price: 'Custom pricing',
-        priceSubtitle: 'w/ fixed terms',
-        features: [
-            'Unlimited everything',
-            'SAML SSO',
-            'Custom MSA',
-            'Dedicated support',
-            'Personalized onboarding & training',
-            'Advanced permissions & audit logs',
-        ],
-        CTAText: 'Get in touch',
-        CTALink: '/contact-sales',
-    },
-]
+const getPlanSummary = (posthog): PlanData[] => {
+    return [
+        {
+            title: 'Totally free',
+            price: 'Free',
+            priceSubtitle: '- no credit card required',
+            features: [
+                'Generous usage limits on all products',
+                'Basic product features',
+                '1 project',
+                '1 year data retention',
+                'Community support',
+            ],
+        },
+        {
+            title: 'Ridiculously cheap',
+            price: '$0',
+            features: [
+                'Generous free tier on all products',
+                'Advanced product features',
+                posthog?.getFeatureFlag?.('two-project-limit') ? '2 projects' : '10 projects',
```

Comment:
this will likely have a delay, there's a `<RenderInClient>` component you can use to give it a placeholder so the default value doesn't show. 

Usage is like:

```js

<RenderInClient
    render={() => {
        return posthog?.getFeatureFlag?.('homepage-billboard-truck') === true ? (
            <BillboardTruck leftHandDrive />
        ) : (
            <BillboardTruck />
        )
    }}
    placeholder={
        <div>
            <Spinner className="w-7 h-7 mx-auto my-12" />
        </div>
    }
/>

```

I'd prob just leave blank for placeholder.

Reply:
Ah I was looking for the component for that. Thank you, will update.